CodeReview/src/nodes/ReviewComment.h

```diff
@@ -46,14 +46,23 @@ class CODEREVIEW_API ReviewComment : public Super<Model::CompositeNode>
 {
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(ReviewComment)
 
-	ATTRIBUTE(Model::Text, date, setDate)
 	ATTRIBUTE(Comments::CommentNode, commentNode, setComment)
 
 	public:
-		ReviewComment(Comments::CommentNode* commentNode, Model::Text* date, Model::Node* parent=nullptr);
+		ReviewComment(Comments::CommentNode* commentNode, qint64 date, Model::Node* parent=nullptr);
+		qint64 date();
```

Comment:
hmm, you can have an `Integer` attribute which holds an `int` You have two choices, either store the number as a string, or store it as two integers. Either way make these private attributes (look at how enumerations are defined for info on private attributes, e.g. method kind in method), since you don't want them to be accessed directly, then make yourself a nice getter/setter that transparently converts to and from `qint64`.

Whatever the case, we need to store the date, so it needs to be represented as attributes.